src/main/java/com/sergiofah/model/Product.java

```diff
@@ -1,11 +1,17 @@
 package com.sergiofah.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import javax.persistence.*;
-import java.io.Serializable;
 
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 @Entity
 @Table(name = "product")
-public class Product implements Serializable {
+public class Product{
```

Comment:
The left brace must always be separated from the previous character by most Java code conventions and our convention.